waterlog/src/App.scss

```diff
@@ -112,23 +112,66 @@ button {
 	color: $trout;
 }
 
-@media screen and (min-width: 700px) {
-	body {
-		background-color: $wild-sand;
-	}
+.usage-header {
+	display: none;
+}
+
+.status{
+	display:none;
 }
 
-@media screen and (min-width: 1400px) {
+@media screen and (min-width: 700px) {
 	body {
-		background-color: $background-color;
+		background-color:$background-color;
 	}
 
-	//TODO: implementing deskop designs
 	/* **************** Home screen wastage ************************** */
+	.wastage-tank-wrapper {
+		display: flex;
+	  margin: 0 auto;
+	}
+
+  .tank img {
+		height: 138px;
+		width: 100px;
+	}
+
+	.tank-container {
+    flex: 4;
+	}
+
+	.alerts-header{
+		display: none;
+	}
+
+  .status{
+		display:block;
+	}
+	
+	.tabcontent{
+		width:80%;
+		margin:0px auto;
+	}
+
+	.map-icon-div-layer2{
+    left:80%;
```

Comment:
Fix this entire file's indentation

Reply:
that page is fine am using size 2 tab will show you the screen shot